coriolis/tasks/migration_tasks.py

```diff
@@ -5,284 +5,92 @@
-        schemas.validate_value(
-            {"volumes_info": volumes_info},
-            schemas.CORIOLIS_DISK_SYNC_RESOURCES_INFO_SCHEMA)
-        LOG.info("Volumes info is: %r" % volumes_info)
-
-        image_paths = [i.get("disk_image_uri") for i in volumes_info]
-        if None in image_paths:
-            raise exception.InvalidActionTasksExecutionState(
-                "disk_image_uri must be part of volumes_info for"
-                " standard migrations")
-
-        target_conn_info = base.unmarshal_migr_conn_info(
-            instance_deployment_info["disk_sync_connection_info"])
-        manager.copy_disk_data(
-            target_conn_info, volumes_info, event_handler)
+class DeployMigrationSourceResourcesTask(
+        replica_tasks.DeployReplicaSourceResourcesTask):
+    pass
 
-        return task_info
 
+class DeployMigrationTargetResourcesTask(
+        replica_tasks.DeployReplicaTargetResourcesTask):
+    pass
 
-class DeleteDiskCopyResources(base.TaskRunner):
-    def run(self, ctxt, instance, origin, destination, task_info,
-            event_handler):
-        provider = providers_factory.get_provider(
-            destination["type"], constants.PROVIDER_TYPE_IMPORT, event_handler)
-        connection_info = base.get_connection_info(ctxt, destination)
-        instance_deployment_info = task_info.get(
-            "instance_deployment_info", {})
-        provider.delete_disk_copy_resources(
-            ctxt, connection_info, instance_deployment_info)
 
-        if instance_deployment_info.get("disk_sync_connection_info"):
-            del instance_deployment_info["disk_sync_connection_info"]
-        if instance_deployment_info.get("disk_sync_tgt_resources"):
-            del instance_deployment_info["disk_sync_tgt_resources"]
+class CreateInstanceDisksTask(
+        replica_tasks.DeployReplicaDisksTask):
+    pass
 
-        return task_info
 
+class CleanupInstanceStorageTask(replica_tasks.DeleteReplicaDisksTask):
+    pass
 
-class FinalizeImportInstanceTask(base.TaskRunner):
-    def run(self, ctxt, instance, origin, destination, task_info,
-            event_handler):
-        provider = providers_factory.get_provider(
-            destination["type"], constants.PROVIDER_TYPE_IMPORT, event_handler)
-        connection_info = base.get_connection_info(ctxt, destination)
-        instance_deployment_info = task_info["instance_deployment_info"]
 
-        result = provider.finalize_import_instance(
-            ctxt, connection_info, instance_deployment_info)
-        if result is not None:
-            task_info["transfer_result"] = result
-        else:
-            LOG.warn(
-                "'None' was returned as result for Finalize Import Instance "
-                "task '%s'.", task_info)
+class FinalizeInstanceDeploymentTask(
+        replica_tasks.FinalizeReplicaInstanceDeploymentTask):
+    pass
 
-        return task_info
 
+class CleanupFailedDeploymentTask(
```

Comment:
Could you please rename this class to `CleanupFailedInstanceDeploymentTask` to be consistent with the rest?